lib/ansible/modules/cloud/ovirt/ovirt_host.py

```diff
@@ -344,13 +344,20 @@ def post_reinstall(self, host):
             timeout=self.param('timeout'),
         )
 
+    def raise_host_exception(self):
+        events = self._connection.system_service().events_service().list(max=20)
```

Comment:
To make it reliable, please record the time when the module started and pass it here, so we always list only relevant events.